compiler/rustc_const_eval/src/interpret/place.rs

```diff
@@ -391,7 +393,16 @@ where
         layout: TyAndLayout<'tcx>,
     ) -> MPlaceTy<'tcx, M::Provenance> {
         assert!(layout.is_sized());
-        self.ptr_with_meta_to_mplace(ptr, MemPlaceMeta::None, layout)
+        self.ptr_with_meta_to_mplace(ptr, MemPlaceMeta::None, layout, /*unaligned*/ false)
+    }
+
+    pub fn ptr_to_mplace_unaligned(
```

Comment:
@tiif this method should also come handy in https://github.com/rust-lang/miri/pull/3852 to deal with the alignment problem in eventfd

Reply:
Yup, thanks!